openwisp_users/admin.py

```diff
@@ -385,16 +385,25 @@ def delete_selected_overridden(self, request, queryset):
 
     def get_inline_instances(self, request, obj=None):
         """
-        Avoid displaying inline objects when adding a new user
+        1. Avoid displaying inline objects when adding a new user
+        2. Make OrganizationUserInline readonly for non-superuser
         """
         if obj:
-            return super().get_inline_instances(request, obj)
+            inlines = super().get_inline_instances(request, obj).copy()
+            if not request.user.is_superuser:
+                for inline in inlines:
+                    if isinstance(inline, OrganizationUserInline):
+                        orguser_index = inlines.index(inline)
+                        inlines.remove(inline)
+                        orguser_readonly = OrganizationUserInlineReadOnly(
+                            self.model, self.admin_site
+                        )
+                        inlines.insert(orguser_index, orguser_readonly)
+                        break
+            return inlines
         inline = OrganizationUserInline(self.model, self.admin_site)
         if request:
-            if hasattr(inline, '_has_add_permission'):
-                has_add_perm = inline._has_add_permission(request, obj)
```

Comment:
I looked as far back as Django 1.8 and I don't see `_has_add_permission`. Is this a relic from the far past or am I misunderstanding something?

Moreover, none of the tests seem to fail on removing this. So, I have removed this for asking the same, please let me know if this needs to be added again!

Reply:
Looking at the files in that version, it seems we're not supporting those django versions anymore:
https://github.com/openwisp/openwisp-users/blob/afeac5527599e7efc97706afd0707d33f14bf27c/.travis.yml#L10-L12
Could be removed but I would do that in a separate commit or PR.